services/crates/crates-dependents.service.js

```diff
@@ -0,0 +1,49 @@
+import Joi from 'joi'
+import { pathParams } from '../index.js'
+import { metric } from '../text-formatters.js'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseCratesService, description } from './crates-base.js'
+
+const dependentsResponseSchema = Joi.object({
+  meta: Joi.object({
+    total: nonNegativeInteger,
+  }).required(),
+}).required()
+
+export default class CratesDependents extends BaseCratesService {
+  static category = 'other'
+  static route = { base: 'crates/dependents', pattern: ':crate' }
+
+  static openApi = {
+    '/crates/dependents/{crate}': {
+      get: {
+        summary: 'Crates.io Dependents',
+        description,
+        parameters: pathParams({
+          name: 'crate',
+          example: 'tokio',
+        }),
+      },
+    },
+  }
+
+  static render({ dependentCount }) {
+    return {
+      label: 'dependents',
+      message: metric(dependentCount),
+      color: dependentCount === 0 ? 'orange' : 'brightgreen',
+    }
+  }
+
+  async fetch({ crate }) {
```

Comment:
Endpoint is `/api/v1/crates/:crate/reverse_dependencies`, so `fetch` had to be implemented manually as `BaseCratesService.fetch` fetches the package meta.

Reply:
agreed :+1: